doc/experimental/phase2.md

```diff
@@ -0,0 +1,137 @@
+    ```
+5. If the target node is a control plane node, stop or shutdown the target node host machine. This will make the following kubectl operations more responsive as the target node status may transit to NotReady after reset:
+    ```
+    kubectl get nodes
+    NAME                        STATUS                        ROLES           AGE    VERSION
+    cz-ocne-control-plane-001   NotReady,SchedulingDisabled   control-plane   106m   v1.26.6+1.el8
+    cz-ocne-control-plane-002   Ready                         control-plane   103m   v1.26.6+1.el8
+    cz-ocne-worker-001          Ready                         <none>          100m   v1.26.6+1.el8
+    cz-ocne-worker-002          Ready                         <none>          106m   v1.26.6+1.el8
+    ```
+6. Generate the ignition file by running CLI
+The CLI cluster join command will display two messages that have the cert-key and token string.
+You must use those values in section 7a and 7b.Also be sure to create run those commands on a 
+control plane node different from the target node.
+
+   6a. If the target node is a **control plane node**, and the control-plane.ign is never generated or the control-plane.ign was generated over two hours ago
+    ```
+    # for control plane node
+    ocne cluster join -c ~/.ocne/byo.yaml -k ~/.kube/kubeconfig.ocne1x -n 1 > control-plane.ign
+    ```
+   
+   6b. If the target node is a **worker node**, and the worker.ign is never generated or the worker.ign was generated over two hours ago
+    ```
+    # for worker node
+    ocne cluster join -c ~/.ocne/byo.yaml -k ~/.kube/kubeconfig.ocne1x -w 1 > worker.ign
+    ```
+7. Generate the join token and upload certs in case of control plane as per instruction printed from the previous step of running "ocne cluster join"
+
+   7a. If the target node is a **control plane node**, and the commands were never executed or executed over two hours ago
+    ```
+    # The [control-plane-node] could be any of the control-plane-nodes other than the target node
+    echo "chroot /hostroot kubeadm init phase upload-certs --certificate-key **** --upload-certs" | ocne cluster console --node [control-plane-node]
+    echo "chroot /hostroot kubeadm token create ****" | ocne cluster console --node [control-plane-node]
+    ```
+   
+   7b. If the target node is a **worker node**, and the commands were never executed or executed over two hours ago
+    ```
+    kubeadm token create ****
+    ```
+
+## Upgrade the node
+
+1. Update the compute instance for the target node to boot with Oracle Cloud Native Environment 2.0 image that has the same kubernetes version and the ignition configuration, control-plane.ign or worker-ign, generated.
+
+   1a. Replace the boot volume with the compatible OCK image.
+
+   1b. Add an item of the Metadata user_data with base64 encoded ignition. Push the Save button. If the instance is running, pushing the Save button will reboot the instance.  
```

Comment:
This is OCI specific plus we might want to show the OCI panes.  Maybe we use a tabbed pane here (future PR)